mappings/net/minecraft/world/gen/UniformIntDistribution.mapping

```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_5428 net/minecraft/world/gen/UniformIntDistribution
+	FIELD field_25809 CODEC Lcom/mojang/serialization/Codec;
+	FIELD field_25810 base I
+	FIELD field_25811 spread I
+	METHOD <init> (II)V
+		ARG 1 base
+		ARG 2 spread
+	METHOD method_30314 of (I)Lnet/minecraft/class_5428;
```

Comment:
if you keep this class name, i'd suggest `fixed` which may be a bit more descriptive

Reply:
Maybe `constant`?

Reply:
just keep it `of` then

Reply:
So, should I rename it or not?